### DISCUSSION TITLE
Internal vaultwarden email images won't load from gmail

### DISCUSSION THREAD

**Discussion by DariB123** [dani-garcia/vaultwarden#2049] [Q&A]
I have my vaultwarden internally facing for security reasons. However, I'd still like to use SMTP for 2FA.

The issue I come across is that gmail proxies the images (which it obviously can't reach since my vaultwarden isn't external). This means that the images look like this:

![image](https://user-images.githubusercontent.com/78615262/138001237-66253624-d611-40f7-aaa4-98f35a677b36.png)

You can see what gmail is trying to do here:
![image](https://user-images.githubusercontent.com/78615262/138001337-0b0e3f71-d419-4f5d-8f6a-d0cf26b44204.png)

Is there any way to get around this?

Thanks in advance

**Comment by iHaveAstream** [comment]
Hi there, 

I'm just about to setup my self-hosted instance of latest Vaultwarden inside Docker and have the same issue.
I think it's rather something "cosmetic" but would be nice if there's a solution.

Thanks!

**Comment by BlackDex** [comment]
Well, the mail templates use images hosted by the system. If you only want to have it internal, then either modify the mail templates so that the images point to a different location, but that would mean if there are updates to the templates, you have to do those manually.

I'm not sure if we want to embed those images or add them as attachments.
